example/fine_advection/advection_driver.cpp

```diff
@@ -146,9 +143,8 @@ TaskCollection AdvectionDriver::MakeTaskCollection(BlockList_t &blocks, const in
       }
     }
 
-    auto set_flx = parthenon::AddFluxCorrectionTasks(
-        start_flxcor | flx | flx_fine | vf_dep, tl, mc0, pmesh->multilevel);
-
+    auto set_flx = parthenon::AddFluxCorrectionTasks(flx | flx_fine | vf_dep, tl, mc0,
```

Comment:
Were these removed `StartReceive` tasks just accidental leftovers from a previous changeset?

Reply:
I removed the `StartReceive` tasks during debugging because they are not necessary for coalesced comms and I was in a state of heightened paranoia. They could be left in there, but they are not required for the old style communication and are no-ops for the coalesced communication.

Reply:
OK if they're not required for any case makes sense to me to remove them. I like to think of the examples as "best practices" that can be safely copy/pasted by downstream code developers

Reply:
I actually wonder if we should maybe remove them from the API? Since the comms machinery got rewritten/unified with packing, they didn't really do anything anyway. (Of course that can be a question for another time. Kind of orthogonal to this MR.)

Reply:
They still allow for pre-posting receives when using non-coalesced communication.